.github/ISSUE_TEMPLATE/webtestplan.md

```diff
@@ -25,27 +25,33 @@ For main, test with a role that has access to all resources.
 - [ ] Verify that Collapse/Expand works and collapsed has icon `>`, and expand has icon `v`
 - [ ] Verify that it automatically expands and highlights the item on page refresh
 
+### Unified Resources
+- [ ] Verify that scrolling to the bottom of the page renders more resources
+- [ ] Verify that all resource types are visible if no filters are present
 #### Servers aka Nodes
 - [ ] Verify that "Servers" table shows all joined nodes
 - [ ] Verify that "Connect" button shows a list of available logins
 - [ ] Verify that "Hostname", "Address" and "Labels" columns show the current values
 - [ ] Verify that "Search" by hostname, address, labels works
```

Comment:
are we keeping these? otherwise this section needs udpating?

Reply:
i made a more generic task in the unified section and removed these. thanks!